programs/compressed-token/src/process_transfer.rs

```diff
@@ -202,7 +202,7 @@ pub fn add_token_data_to_input_compressed_accounts(
 // TODO: consider moving this function to helpers
 /// Get static cpi signer seeds
 pub fn get_cpi_signer_seeds() -> [&'static [u8]; 2] {
-    let bump: &[u8; 1] = &[255];
+    let bump: &[u8; 1] = &[254];
     let seeds: [&'static [u8]; 2] = [b"cpi_authority", bump];
     seeds
 }
```

Comment:
why the change?

Reply:
different program id different derivation